### DISCUSSION TITLE
Script as part of batch operation!

### DISCUSSION THREAD

**Discussion by Abhi-SimuTech** [ansys/pyaedt#5030] [Q&A] [answered]
Hi All,

We have a situation where we will like to extract certain antenna/far field parameters from the last adaptive pass of a HFSS simulation after simulation completes. With the examples from PyAEDT documentation page as well as drawing from some of the discussions here, we can possibly figure out how to export the desired FF quantities in our desired format. But, we have the simulations running in batches on a remote server. So, in this case, how to make this script part of the batch operation that happens at the conclusion of the simulation? Or, is there a better method that you would suggest.

Look forward to your thoughts on this. Thanks in advance.

Regards

**Comment by Samuelopez-ansys** [comment, marked as answer]
@Abhi-SimuTech If your workflow is about antenna farfield data, I would suggest to run in the server a simple PyAEDT script that exports the data in the server:

https://aedt.docs.pyansys.com/version/stable/API/Visualization.html#farfield

Then you can grab the data to the client side, to perform the post-processing. This class allows offline post-processing of Farfield (no AEDT need).

About launching this script when the solver finishes, I would suggest creating an Ironptyhon script that launches the CPython script. This is what the PyAEDT installer buttons are doing, here you can find the Run PyAEDT Script code:

https://github.com/ansys/pyaedt/blob/main/pyaedt/workflows/templates/run_pyaedt_script.py_build

**Reply by Abhi-SimuTech** [reply]
Thanks for the helpful directions.
